common/app/mvt/MultiVariateTesting.scala

```diff
@@ -19,78 +17,67 @@ import conf.switches.Switches.ServerSideTests
 // }
 
 object ABOpenGraphOverlay extends TestDefinition(
-  variants = Nil,
   name = "ab-open-graph-overlay",
   description = "Add a branded overlay to images cached by the Facebook crawler",
   sellByDate = new LocalDate(2016, 6, 29)
 ) {
   override def isParticipating(implicit request: RequestHeader): Boolean = {
-    request.queryString.get("page").exists(_.contains("facebookOverlayVariant")) && switch.isSwitchedOn && ServerSideTests.isSwitchedOn
+    request.queryString.get("page").exists(_.contains("facebookOverlayVariant")) && super.isParticipating(request)
```

Comment:
also ref my previous comment people will surely forget to call super.isParticipating sometimes